src/components/Skills.js

```diff
@@ -0,0 +1,81 @@
+import { useState } from "react";
+
+//import data from object
+import SoftSkills from "../data/soft-skills.json";
+import HardSkills from "../data/hard-skills.json";
```

Comment:
Ai putea să faci un singur fișier de forma:

```json
{
    "Soft skills": [
    ],
    "Hard skills": [
    ]
}
```

Cred că ar face această pagină mai flexibilă, cu mai puțin cod duplicat.